plugins/synaptics-cape/fu-synaptics-cape-device.c

```diff
@@ -70,10 +73,11 @@ typedef enum {
 	FU_SYNAPTICS_CMD_GET_VERSION = 0x103,		  /* gets cur firmware version */
 } FuCommand;
 
-/* CAPE Fuupd device structure */
+/* CAPE Fwupd device structure */
 struct _FuSynapticsCapeDevice {
 	FuHidDevice parent_instance;
 	guint32 ActivePartition; /* active partition, either 1 or 2 */
+	gboolean InReportViaInterrupt; /* host can get hid report via INTERRUPT IN EP*/
```

Comment:
maybe call this in_report_via_interrupt?

Reply:
I am okay with it, but I am a little confused. Last time, I was told to change variable 'active_partition' from snake case to camel case.

Reply:
Yes, my fault. What I meant was the variable name should be active_partition and the printable debug _title_ should be ActivePartition. No biggie.

Reply:
Okay, I got it. will update both variables to snake case.